hardhat.config.ts

```diff
@@ -446,6 +465,12 @@ const config: HardhatUserConfig = {
         network: 'scroll-goerli',
         deployment: 'usdc',
         auxiliaryBase: 'goerli'
+      },
+      {
+        name: 'scroll',
```

Comment:
`scroll-usdc`

Reply:
this config info is the same for all markets on Scroll mainnet, not sure why `usdc` needs to be specified?

Reply:
To differentiate between future Scroll markets like `scroll-weth`. Each Comet, even if on the same network, has a different config.

Reply:
I'm a little confused as that's not how all the other networks here are configured. They're listed by network and not market. For example `scroll-goerli` is not the Goerli market on Scroll but rather the Scroll testnet bridged to goerli, and there are multiple other networks listed like this too, with mainnet markets simply listed as the name of the network.

Reply:
If you're looking at `mainnet` and `goerli`, those are legacy conventions. Take a look at `arbitrum-usdc` and `base-weth` for example.

The network name is defined in the `network` field.